moveit_ros/hybrid_planning/hybrid_planning_manager/hybrid_planning_manager_component/include/moveit/hybrid_planning_manager/hybrid_planning_manager.h

```diff
@@ -54,14 +54,14 @@ namespace moveit::hybrid_planning
 /**
  * Class HybridPlanningManager - ROS 2 component node that implements the hybrid planning manager.
  */
-class HybridPlanningManager : public rclcpp::Node
```

Comment:
Inheriting from `enable_shared_from_this` might be worse than inheriting from `Node`. Why do you need to do this?

Reply:
There is a way to get around that clang-tidy check: https://github.com/ros-planning/moveit2/blob/2de48a7aafacd45d4f9d4c667ff7337b1da31ddd/moveit_ros/moveit_servo/include/moveit_servo/servo_node.h#L53

```
if (!planner_logic_instance_->initialize(HybridPlanningManager::shared_from_this()))
```
This is a part of the Hybrid planner design I'd like to find a way to get rid of. This circular relationship between the manager and the planners seems unnecessary to me.